plugins/node/opentelemetry-instrumentation-hapi/tsconfig.json

```
@@ -1,11 +1,12 @@
 {
-    "extends": "../../../tsconfig.base",
-    "compilerOptions": {
-      "rootDir": ".",
-      "outDir": "build"
-    },
-    "include": [
-      "src/**/*.ts",
-      "test/**/*.ts"
-    ]
-  }
+  "extends": "../../../tsconfig.base",
+  "compilerOptions": {
+    "rootDir": ".",
+    "outDir": "build",
+    "skipLibCheck": true
```

```suggestion
    "outDir": "build"
```
should still compile IIUC :slightly_smiling_face:

---

Which means we can remove all changes to this file. The only other changes were indentation.

---

I reverted all changes to this file.